src/index.ts

```diff
@@ -16,3 +17,8 @@ app.listen(
     logger.info(`Server running on http://${host}:${port}`);
   },
 );
+
+startJobQueue().catch((err) => {
+  logger.error('Job queue failed to start');
+  logger.error(err);
```

Comment:
Can these be combined into a single call to `logger.error`? I believe we want to keep such information together, so it doesn't get interleaved with other unrelated logs.

Reply:
I'll look into it -- I maybe conflated memories but I know at least SOME logger did not take arbitrary numbers of parameters.

Reply:
This was a relevant thread -- https://github.com/pinojs/pino/issues/161